Tools/cameras_gimbals/siyi-download/siyi-download.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
+"""
+Downloads files from a Siyi camera connected via ethernet
+"""
```

Comment:
```suggestion

AP_FLAKE8_CLEAN
"""
```

Reply:
thanks, added AP_FLAKE8_CLEAN.  I removed it because it was showing up in the usage somehow